### PR TITLE
Fix for error handling on set command in admin cli.

### PR DESCRIPTION
**Pull request by govint**
Modified vmdk_ops.py set_vol_opts() to not raise validation errors, surrounding code doesn't raise errors and instead log the errors as warnings. This was the only case where an exception was being raised and un-handled. Plus ValidationError is specific to vmdk_ops.py. 

Test is already there for vmdkops_admin.py and vmdk_ops.py for access option.

Code now prints the error and the usage:

vmdkops_admin.py set --options="acces=read-write" --volume=testvol-11@bigone

WARNING:root:Invalid options: ['acces']
Options that can be edited: ['attach-as', 'access']
Failed to update acces=read-write for testvol-11@bigone.

vmdkops_admin.py set --options="access=read-write" --volume=testvol-11@bigone
Successfully updated settings for : testvol-11@bigone

### CODE REVIEW

**Comment by govint** [comment]
Fixes #609

File: esx_service/cli/vmdkops_admin.py [hunk 1]
```diff
@@ -563,10 +563,13 @@ def status(args):
...
-        print('Successfully updated settings for : {0}'.format(args.volume))
-    else:
+    try:
+        set_ok = vmdk_ops.set_vol_opts(args.volume, args.options) 
+        if set_ok:
+           print('Successfully updated settings for : {0}'.format(args.volume))
+        else:
+           print('Failed to update {0} for {1}.'.format(args.options, args.volume))
+    except Exception as ex:
         print('Failed to update {0} for {1}.'.format(args.options, args.volume))
```
Comment (pdhamdhere) [inline esx_service/cli/vmdkops_admin.py]: print ex.message so user knows cause of failure.

Comment (msterin) [inline esx_service/cli/vmdkops_admin.py]: +1 on this. 
`WARNING:root:Invalid options: ['acces']` seems to be showing in the log, not sysout. Or is it in sysout ? 
Can you clarify the exact print happening on typos here?

**Comment by govint** [comment]
The error messages are all on the command line. I'll print the exception
message if any.

**Comment by msterin** [comment]
@Govint - can you please add the current error message (as printed by Admin CLI on this error, and some other error for comparison) so we can LGTM and merge this one ?

**Comment by govint** [comment]
Updated the change to ensure that exceptions are caught and the exception message is printed

1. Exception handled and error printed on stdout.

[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py set --options="acces=read-write" --volume=testvol-11@bigone
Failed to update acces=read-write for testvol-11@bigone - "Invalid options: ['acces'] \nOptions that can be edited: ['attach-as', 'access']".

2. When the correct option is used
[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py set --options="access=read-write" --volume=testvol-11@bigone
Successfully updated settings for : testvol-11@bigone

3. A missing volume
[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py set --options="access=read-write" --volume=testvol-110@bigone
WARNING:root:Volume testvol-110 not found.
Failed to update access=read-write for testvol-110@bigone.

**Comment by msterin** [comment]
Fixes #609

**Pull request merged**